src/components/reservation/ReservationModalBody.tsx

```diff
@@ -58,6 +58,14 @@ const ReservationModalBody = ({
   parallelManifestations,
   selectedPeriodical
 }: ReservationModalProps) => {
+  const t = useText();
+  const config = useConfig();
+  const branches = config<AgencyBranch>("branchesConfig", {
+    jsonParse: true
+  });
+
+  const userListItemsBranches = Array.isArray(branches) ? branches : [];
```

Comment:
Consider if you can provide a more explanatory name.

I do not understand why it is called `userListItemsBranches`.